hooks/playbooks/manila_create_default_resources.yml

```diff
@@ -8,7 +8,31 @@
     extra_specs:
       snapshot_support: "True"
       create_share_from_snapshot_support: "True"
+    manila_provider_network_name: storage
+    manila_provider_network_vlan: 21
```

Comment:
I have very little context for this, but did you want the vlan to be 21 or 24 (as per the commit msg)?
And same for the range -  172.18.0.0/24 or 172.21.0.0/24?

Reply:
Thanks for the review.

I actually want to default to the usual storage network which is 21 in case the NFS network is not used. So gamma will use the storage network while delta will override it to the NFS network. 
```
Variables manila_provider_network_{name,vlan,start,end,range}
default to the storage network, but can be overridden to the
NFS network within a CI job definition.
```